### ISSUE TITLE
Stuck on ansible [consul | restart consul ] after adding more worker nodes.

### ISSUE THREAD

**Issue by zoza1982** [CiscoCloud/microservices-infrastructure#1068]
If I decide for given terraform cluster to add more worker nodes  and run ansible playbook again after adding more node....it will get  stuck on consul restart...with not being able to proceed nor timeout will kick in.

Below in screenshot I am re-running terrform.yml again to get new nodes worker002/03 in the cluster.
I tried this several time. Same issue every time.

```
TASK: [consul | deploy acl bootstrap] ***************************************** 
ok: [mi-edge-01]
ok: [mi-worker-001]
ok: [mi-control-02]
ok: [mi-control-03]
ok: [mi-control-01]
changed: [mi-worker-003]
changed: [mi-worker-002]

TASK: [consul | upsert agent acl] ********************************************* 
changed: [mi-edge-01]

TASK: [consul | create agent_acl consul config] ******************************* 
ok: [mi-edge-01]
ok: [mi-control-03]
ok: [mi-control-02]
ok: [mi-worker-001]
ok: [mi-control-01]
changed: [mi-worker-003]
changed: [mi-worker-002]

NOTIFIED: [consul | restart consul] ******************************************* 

```

If I do fresh installation , works great. 
I am testing this on Cisco Cloud ( Openstack)

**Comment by zoza1982** [comment]
Also to mention , I am tunneling trough bastion hosts with ansible to reach hosts in openstack private network . I did noticed some weird behavior when running for example upgrade packages yml , it would not recognize that machine came back online after reboot.

I had to modify upgrade-packages.yml to get past trough never being able to recognize if hosts came back up.

```
  - name: wait for host boot
    local_action:
      module: wait_for
      host: "{{ ansible_ssh_host }}"
      #port: 22
      delay: "{{ boot_wait | default(100) }}"
      timeout: 120
      state: started
    when: reboot.changed
```

I commented port and changed boot wait default to 100 sec.

**Comment by ryane** [comment]
this should be fixed by #1070. please re-open if there are still issues
